paddle/fluid/framework/parallel_executor.cc

```diff
@@ -770,10 +780,24 @@ ParallelExecutor::ParallelExecutor(const std::vector<platform::Place> &places,
             exec_strategy, member_->local_scopes_, member_->local_exec_scopes_,
             member_->places_, graph));
       } else {
-        VLOG(3) << "use FastThreadedSSAGraphExecutor";
-        member_->executor_.reset(new details::FastThreadedSSAGraphExecutor(
-            exec_strategy, member_->local_scopes_, member_->local_exec_scopes_,
-            member_->places_, graph));
+#if defined(PADDLE_WITH_XPU)
```

Comment:
I think this defined can be removed for now.

Reply:
done